.github/workflows/releases.yml

```diff
@@ -46,7 +34,7 @@ jobs:
           github_token: ${{secrets.GITHUB_TOKEN}}
           workflow: build.yml
           workflow_conclusion: success
-          branch: develop
+          commit: ${{github.event.push.head.sha}} # note: by the time this workflow runs, we expect the tagged commit build has already succeeded
```

Comment:
d: what would happen if it is still running? Wouldnt this then use the wrong commit as the release commit?

Reply:
this is basically a precondition that we have to manually ensure, but given that this is a manual process anyway I expect this is not a problem